WcaOnRails/app/jobs/sync_mailing_lists_job.rb

```diff
@@ -5,6 +5,7 @@ class SyncMailingListsJob < ApplicationJob
 
   def perform
     GsuiteMailingLists.sync_group("delegates@worldcubeassociation.org", User.delegates.map(&:email))
+    GsuiteMailingLists.sync_group("delegates.africa@worldcubeassociation.org", User.african_delegates.map(&:email))
```

Comment:
Maybe something like this instead, actually?

```ruby
region_query_by_mailing_list = {
  'delegates.africa@worldcubeassociation.org' => '%Africa%',
}

region_query_by_mailing_list.each |mailing_list, region_query| do
  senior_delegate = User.senior_delegate.where("region_id LIKE #{region_query}")
  delegates = senior_delegate + senior_delegate.subordinate_delegates
  GsuiteMailingLists.sync_group(mailing_list, delegates)
end
```

Reply:
I went ahead with this!